### ISSUE TITLE
Attribute selection on web-app

### ISSUE THREAD

**Issue by codyemoffitt** [arpachuilo/nemo#19]
1. Need to be able to select which attributes to feed into the QueryBuilder, based on question parameters and patient demographic data.
2. Example:
   Question with one parameter:
   Relevant Attributes from patient_dimension:
   - vital_status_cd
   - birth_date
   - death_date
   - sex_cd
   - age_in_years_num
   - language_cd
   - race_cd
   - marital status_cd
   - religion_cd
   - zip_cd
   - statecityzip_path
   - income_cd
     Relevant attributes from observation_fact (duplicate these all for each parameter):
   - concept_cd
   - start_date
   - modifier_cd
   - instance_num
   - valtype_cd
   - tval_char
   - nval_num
   - valueflag_Cd
   - quantity_num
   - units_cd
   - end_date
   - confidence_num

So basically, in the website they can select which attributes land in the dataset for the learners.
This can be stored as a JSON string in the Question table, then the query builder can read it, and only SELECT those attributes.

Need a unique naming scheme for the observation_fact columns, for when there are more than one parameter in the question.
Example:
o1concept_cd, 01..., o2concept_cd, o2....,
Another, maybe better plan is to use the concept code as part of the attribute name, so order doesn't matter:
'401.9concept_cd', '401.9start_date', ..., '301.4conceptcd', '301.4start_date'

This engine will also be useful when determining what attributes/input boxes are needed to define the prediction patient.
